npm/index.js

```diff
@@ -6,9 +6,11 @@ var pathFile = path.join(__dirname, 'path.txt')
 function getElectronPath () {
   if (fs.existsSync(pathFile)) {
     var executablePath = fs.readFileSync(pathFile, 'utf-8')
```

Comment:
`executablePath` should be moved out of the scope since it is used by other scopes, we should not rely on JS"s weird rules for undefined variables.

```js
var executablePath
if (...) {
  executablePath = ... 
}
```

Reply:
Done, and rebased.